src/Api/Modules/PotStageHistory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Zoho\CRM\Api\Modules;
+
+class PotStageHistory extends AbstractRecordsModule
+{
+    protected static $primary_key = 'POTENTIALSTAGEHISTORYID';
+
+    protected static $associated_entity = \Zoho\CRM\Entities\PotStageHistory::class;
+
+    protected static $supported_methods = [
+        'getRelatedRecords',
+    ];
+
+    public function getPotentialHistory($potentialId)
```

Comment:
Variable naming: you must replace `$potentialId` with `$potential_id`.
This naming convention could change in a future release, but at least for now we shall stick to the same thing everywhere.